features/com.gwtplugins.eclipse.sdkbundle.gwt211.feature/feature.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="com.gwtplugins.eclipse.sdkbundle.gwt211.feature"
+      label="%featureName"
+      version="2.11.0.qualifier"
+      provider-name="%providerName">
+
+   <description>
+      %description
+   </description>
+
+   <copyright>
+      Copyright 2015 to 2017
+   </copyright>
+
+   <license url="%licenseURL">
+      %license
+   </license>
+
+   <url>
+      <update label="GWT Eclipse Plugin Update Site" url="http://storage.googleapis.com/gwt-eclipse-plugin/v3/release"/>
```

Comment:
This URL hasn't been correct in a while, I think we want to point to something like https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/4.0.0/ instead?